wazo_websocketd/bus.py

```diff
@@ -448,7 +453,7 @@ async def __aenter__(self):
     async def __aexit__(self, *args):
         await self._connection_pool.stop()
 
-    async def create_consumer(self, token: str) -> BusConsumer:
+    async def create_consumer(self, token: dict) -> BusConsumer:
```

Comment:
If possible it would be really cool to type the token dict. As we use a lot of `token` variables and they have tons of different values and it's very confusing. In fact we do it so often that we should probable create these types in xivo-lib-python as they are used in almost all modules.

example:

```python
class TokenMetadataDict(TypedDict):
    uuid: str
    tenant_uuid: str
    purpose: str

class TokenDict(TypedDict):
    session_uuid: str
    metadata: TokenMetadataDict
    acl: list[str]
```

Reply:
As discussed together, the TokenDict typing is done inside wazo-auth-client